hazelcast/src/main/java/com/hazelcast/concurrent/atomiclong/AtomicLongProxy.java

```diff
@@ -66,6 +67,11 @@ public long addAndGet(long delta) {
     }
 
     @Override
```

Comment:
Could you change the implementation so that the deprecated 'asyncXXX' are forwarding to the new 'XXXAsync'? This way it is a lot easier to get rid of the deprecated methods instead of having more reliance on them.